packages/commonwealth/server/util/denormalizedCountUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { DB } from '@hicommonwealth/model';
+import { Transaction } from 'sequelize';
+
+export const incrementProfileCount = async (
```

Comment:
I noticed the decrement func was incline while the increment is moved into a util file. Maybe both funcs should be in the util file?

Reply:
Good catch, fixing